study-builder/studies/brugada/study-events.conf

```diff
@@ -0,0 +1,435 @@
+        ],
+        "linkedActivityCode": "BACKGROUND_INFORMATION",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["BACKGROUND_INFORMATION"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.six},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 2
+    },
+
+    // GENETIC_TESTING reminder #1 (3 weeks)
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.genetic_testing_and_family_history_reminder_en}, "language": "en", "isDynamic": true }
+        ],
+        "linkedActivityCode": "GENETIC_TESTING",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["GENETIC_TESTING"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.three},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // GENETIC_TESTING reminder #2 - Final (6 weeks)
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.genetic_testing_and_family_history_reminder_en}, "language": "en", "isDynamic": true }
+        ]
+        "linkedActivityCode": "GENETIC_TESTING",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["GENETIC_TESTING"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.six},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 2
+    },
+
+    // GENETIC_TESTING "Completion Thank You " email
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.completion_thank_you_en}, "language": "en", "isDynamic": true }
+        ],
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // Activity Instance Creation
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "LIFESTYLE"
+      },
+      "preconditionExpr": "true",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+//    {
```

Comment:
can we clean up these  commented events if its not needed

Reply:
cleaned up